DetectorDescription/Core/interface/DDUnits.h

```diff
@@ -0,0 +1,31 @@
+#ifndef DETECTOR_DESCRIPTION_DD_UNITS_H
+#define DETECTOR_DESCRIPTION_DD_UNITS_H
+
+#include <cmath>
+
+#define ConvertTo(_x, _y) (_x)/(1.0_##_y)
+
+constexpr long double _pi(M_PI);
+
+constexpr long double operator"" _deg( long double deg )
```

Comment:
I believe all the examples of user defined types I've see have had them defined within a namespace. Perhaps using `namespace dd` would be appropriate?

Reply:
See
https://akrzemi1.wordpress.com/2012/10/29/user-defined-literals-part-iii/